src/ReinforcementLearningZoo/src/algorithms/nfsp/nfsp.jl

```diff
@@ -0,0 +1,190 @@
+"""
+Neural Fictitious Self-Play (NFSP) agent implemented in Julia.
+
+See the paper https://arxiv.org/abs/1603.01121 for more details.
+"""
+export NFSPAgent, NFSPAgents
+
+using Distributions: TruncatedNormal
+
+mutable struct NFSPAgents <: AbstractPolicy
+    agents::Dict{Any, AbstractPolicy}
+end
```

Comment:
We don't use plural case `NFSPAgents` to differentiate with `NFSPAgent`.

Since we don't really need to export `NFSPAgent`, we can maybe rename `NFSPAgent` to `NFSPAgentInstance`?